Readme.md

```diff
@@ -1,22 +1,28 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128577426/21.2.3%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T289831)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
+# Pivot Grid for Web Forms - How to Export the Pivot Grid's Data to a File
 
-* [WebForm1.aspx](./CS/ASPPivotGridExport/WebForm1.aspx) (VB: [WebForm1.aspx](./VB/ASPPivotGridExport/WebForm1.aspx))
-* [WebForm1.aspx.cs](./CS/ASPPivotGridExport/WebForm1.aspx.cs) (VB: [WebForm1.aspx.vb](./VB/ASPPivotGridExport/WebForm1.aspx.vb))
-<!-- default file list end -->
-# How to export the pivot grid's data to a file
-<!-- run online -->
-**[[Run Online]](https://codecentral.devexpress.com/t289831/)**
-<!-- run online end -->
 
+This example shows how to export the Pivot Grid's data in the Data-Aware or WYSIWYG [types](https://docs.devexpress.com/AspNet/114650/components/pivot-grid/export/export?p=netframework#export-types) and configure the export options. 
 
-This example shows how to export the pivot grid's data using the Data-Aware or WYSIWYG types and set the export options. To learn more, see <a href="https://documentation.devexpress.com/#AspNet/CustomDocument7260">Export Overview</a>.<br />- Click the <strong>Export to XLSX</strong> button to export data using the Data-Aware type. In the exported XLSX file fixed row headers are disabled and a sheet name is set to "Pivot Grid Export". <br />- Click the <strong>Export to PDF</strong> button to export data using the WYSIWYG type. The Print dialog is displayed when the exported PDF file is opened.
+- Click the **Export to XLSX** button to export data in the Data-Aware type. In the exported XLSX file, fixed row headers are disabled and a sheet name is set to "Pivot Grid Export".

```

Comment:
I understand that Data-Aware and WYSIWYG are "types" (based on the above paragraph and corresponding link), but since you use the preposition "in" here, could we say "mode" instead? For example:

`- Click the **Export to XLSX** button to export data in Data-Aware mode. In the exported XLSX file, fixed row headers are disabled and the sheet name is set to "Pivot Grid Export".`

The reason I suggest "mode" is that I noticed this term used in the following link:
https://docs.devexpress.com/WPF/10018/controls-and-libraries/data-grid/printing-and-exporting/data-aware-export

Reply:
```suggestion
- Click the **Export to XLSX** button to export data in Data-Aware mode. In the exported XLSX file, fixed row headers are disabled and the sheet name is set to "Pivot Grid Export".
```
We've discussed it and come to the conclusion that it is okay to use "mode" here.